libkbfs/folder_branch_ops.go

```diff
@@ -279,6 +279,11 @@ type folderBranchOps struct {
 	// rekey with a paper key prompt, if enough time has passed.
 	// Protected by mdWriterLock
 	rekeyWithPromptTimer *time.Timer
+
+	// latestRemoteMergedRevision tracks the latest know merged revision of on
+	// server
+	muLatestRemoteMergedRevision sync.RWMutex
+	latestRemoteMergedRevision   MetadataRevision
```

Comment:
No need for the word "Remote" here -- it might not even be true if the server got an update since you last heard about it.